test/cpp/util/BUILD

```diff
@@ -104,5 +104,29 @@ cc_test(
     ],
 )
 
-
-
+cc_binary(
+    name = "grpc_cli",
+    srcs = [
+        "cli_call.cc",
+        "cli_call.h",
```

Comment:
https://github.com/grpc/grpc/blob/master/build.yaml#L1320

and 

https://github.com/grpc/grpc/blob/master/build.yaml#L3702